internal/cmd/envs.go

```diff
@@ -611,3 +626,7 @@ func buildUpdateReq(ctx context.Context, client *coder.Client, conf updateConf)
 	}
 	return &updateReq, nil
 }
+
+func autoStartInfo() {
+	clog.LogInfo("⚡NEW: Automate daily environment startup", "Visit /preferences to configure your preferred time")
```

Comment:
Since we know the access url, we _could_ link directly to the dash here.

Reply:
this value is accessible with the `internal/config` package.

Reply:
Thanks! Added here: https://github.com/cdr/coder-cli/pull/236/commits/3cffbca09546735cd0372df8330a3d956d611374